lib/livebook_web/live/session_live.ex

```diff
@@ -63,7 +63,8 @@ defmodule LivebookWeb.SessionLive do
            page_title: get_page_title(data.notebook.name),
            saved_secrets: get_saved_secrets(),
            select_secret_ref: nil,
-           select_secret_options: nil
+           select_secret_options: nil,
+           protocols: Config.allowed_uri_schemes()
```

Comment:
```suggestion
           protocols: Config.allowed_uri_schemes() |> Enum.join(",")
```

Let's also rename `protocols` to `sechemes` everywhere for consistency :)